azure_arc_servers_jumpstart/esu/bicep/host/host.bicep

```diff
@@ -0,0 +1,193 @@
+@description('The name of your Virtual Machine')
+param vmName string = 'ESU-Host'
+
+@description('Username for the Virtual Machine')
+param windowsAdminUsername string = 'arcdemo'
+
+@description('Password for Windows account. Password must have 3 of the following: 1 lower case character, 1 upper case character, 1 number, and 1 special character. The value must be between 12 and 123 characters long')
+@minLength(12)
+@maxLength(123)
+@secure()
+param windowsAdminPassword string
+
+@description('The Windows version for the VM. This will pick a fully patched image of this given Windows version')
+param windowsOSVersion string = '2022-datacenter-g2'
+
+@description('Location for all resources')
+param location string = resourceGroup().location
+
+@description('Resource Id of the subnet in the virtual network')
+param subnetId string
+
+param resourceTags object = {
+  Project: 'jumpstart_servers'
+}
+
+@description('ESU deployment type, could be ws, sql or both')
+param esu string = 'both'
+
+@description('Client id of the service principal')
+param spnClientId string
+
+@description('Client secret of the service principal')
+@secure()
+param spnClientSecret string
+
+@description('Tenant id of the service principal')
+param spnTenantId string
+
+@description('The base URL used for accessing artifacts and automation artifacts.')
+param templateBaseUrl string
+
+@description('Option to disable automatic cluster registration. Setting this to false will also disable deploying AKS and Resource bridge')
+param registerCluster bool = true
+
+@description('Choice to deploy Bastion to connect to the client VM')
+param deployBastion bool = false
+
+@description('Public DNS to use for the domain')
```

Comment:
why do we need this parameter?

Reply:
It is a flag to access the VM via Bastion, I set it up like this for Internal MSFT use